manifests/service.pp

```diff
@@ -55,17 +55,10 @@
       }
     }
 
-    # firewalld must be disabled on EL7+
-    case $::operatingsystem {
-      'RedHat','CentOS','OracleLinux': {
-        if $::operatingsystemmajrelease > '6' {
-          service{ 'firewalld':
-            ensure => 'stopped',
-            enable => false,
-            before => Service['iptables']
-          }
-        }
-      }
+    # firewalld should be disabled
+    service{ 'firewalld':
+      ensure => 'stopped',
+      enable => false
```

Comment:
why was the before removed?  Looks like before Service[iptables] and before Service[ipv6tables] would still be needed

Reply:
Running this version of iptables in all acceptance tests for which iptables is listed in the fixtures.